packages/theme-default/src/components/HomeFeature/index.tsx

```diff
@@ -9,13 +12,20 @@ const getGridClass = (feature: Feature): string => {
   return `${GRID_PREFIX}${span || 4}`;
 };
 
-export function HomeFeature({ frontmatter }: { frontmatter: FrontMatterMeta }) {
+export function HomeFeature({
+  frontmatter,
+  routePath,
+}: {
+  frontmatter: FrontMatterMeta;
+  routePath: string;
+}) {
   const features = frontmatter?.features;
 
   return (
     <div className="overflow-hidden m-auto flex flex-wrap max-w-6xl">
       {features?.map(feature => {
-        const { icon, title, details, link } = feature;
+        const { icon, title, details, link: rawLink } = feature;
+        const link = normalizeHrefInRuntime(withBase(rawLink, routePath));
```

Comment:
We should use `isExternalUrl` from `@rspress/shared` to determine a url is external or not, then
```ts
const link = isExternalUrl(link) ? link : normalizeHrefInRuntime(withBase(link, routePath))
```